nixos/modules/services/cluster/kubernetes/kubelet.nix

```diff
@@ -235,19 +226,12 @@ in
   ###### implementation
   config = mkMerge [
     (mkIf cfg.enable {
-      services.kubernetes.kubelet.seedDockerImages = [infraContainer];
-
       systemd.services.kubelet = {
         description = "Kubernetes Kubelet Service";
         wantedBy = [ "kubernetes.target" ];
-        after = [ "network.target" "docker.service" "kube-apiserver.service" ];
-        path = with pkgs; [ gitMinimal openssh docker utillinux iproute ethtool thin-provisioning-tools iptables socat ] ++ top.path;
+        after = [ "network.target" "cri-o.service" "kube-apiserver.service" ];
+        path = with pkgs; [ gitMinimal openssh cri-o utillinux iproute ethtool thin-provisioning-tools iptables socat ] ++ top.path;
```

Comment:
Would need to use `config.virtualisation.cri-o.package` for the final wrapper that may include `extraPackages` from the `cri-o` module.

Reply:
Fixed :+1: